lib/hammer_cli/validator.rb

```diff
@@ -98,13 +142,17 @@ def all(*to_check)
     end
 
     def option(to_check)
-      all(to_check)
+      OneOptionConstraint.new(@options, to_check)
```

Comment:
Sure, nothing against that. Just thought it might be a good idea to have a note about this kept somewhere